lib/search/embeddings/paraphrase_albert_small.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Search.Embeddings.ParaphraseAlbertSmall do
+  alias Search.Embeddings
+
+  use Embeddings.Embedding,
+    model: {:hf, "sentence-transformers/paraphrase-albert-small-v2"},
+    embedding_size: 768,
+    serving_opts: [
+      compile: [batch_size: 16, sequence_length: 100],
+      defn_options: [compiler: EXLA]
+    ]
```

Comment:
```suggestion
    ],
    load_mode_opts: [backend: EXLA.Backend]
```

(now the default backend is `{EXLA.Backend, client: :host}`, but the parameters we want to load onto the preferred client, that is, GPU if available)